src/cogs/PrequisiteChecker.py

```diff
@@ -13,8 +15,40 @@ def __init__(self, client):
         self.client = client
 
     @commands.command()
-    async def ping(self, ctx):
-        await ctx.send("Pong!")
+    async def prereq(self, ctx, arg):
+
+        # A lot of error checking and log messages
+        if len(arg) != 7:
+            await ctx.send("Invalid Input")
+            return
+
+        program = arg[0:4]
```

Comment:
It's best if you convert this to upper-case since the CSV is upper-case. Then it no longer becomes case-sensitive for users to invoke.